fiat-roles/src/main/java/com/netflix/spinnaker/fiat/permissions/DefaultPermissionsResolver.java

```diff
@@ -64,6 +64,10 @@
 
   @Override
   public UserPermission resolveUnrestrictedUser() {
+    // Ensure we're going to reload app and service account definitions
+    for (ResourceProvider provider : resourceProviders) {
```

Comment:
Can we invalidate the cache before we start a user role sync instead of in the middle of it ? i.e. somewhere here:
https://github.com/spinnaker/fiat/blob/master/fiat-roles/src/main/java/com/netflix/spinnaker/fiat/roles/UserRolesSyncer.java#L145

Reply:
Sure. Moved to `UserRoleSyncer`.